### PR TITLE
CLOUD-2691 - Change container family name from jboss-eap-7-tech-preview to jboss-eap-7 and use the --tech-preview flag when building

### PR DESCRIPTION
**Pull request by luck3y**
https://issues.jboss.org/browse/CLOUD-2691
https://issues.jboss.org/browse/CLOUD-2898
Signed-off-by: Ken Wills <kwills@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request title is properly formatted: `[CLOUD-XYA] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

**Comment by luck3y** [comment]
Note some additional commits will be added here for tests shortly.

**Comment by luck3y** [comment]
Note the CLOUD-2898 changes made sense to do at the same time here, as the container name had changed etc.

**Comment by luck3y** [comment]
@bstansberry @rhusar This fixes the EAP clustering tests for scale up / down etc, and they currently pass.

File: tests/arq/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Arquillian Tests for JBoss EAP
+
+This project contains OpenShift v3 tests for the JBoss EAP image and
```
Comment (bstansberry) [inline tests/arq/README.md]: and .... ?

**Review by bstansberry — DISMISSED** [review]

File: tests/arq/README.md (hunk 1 above)
Comment (luck3y) [reply tests/arq/README.md]: ... and I was going to fast for my own good ;)

**Review by bstansberry — DISMISSED** [review]

**Comment by luck3y** [comment]
@bstansberry Sorry for the additional change, I decided that dev-overrides was safer having the -tech-preview suffix, and CI will just use image.yaml anyway.

**Comment by rhusar** [comment]
@luck3y can you please point me to the bits that resolve the scaling issues?

**Comment by luck3y** [comment]
@rhusar I'll link it to you in HC, its CR2 of CD 14.

**Review by bstansberry — APPROVED** [review]
Approved but please squash.

**Pull request merged**